try:
    result = 45/5
    # result = 45/0
except:
    print('error happen')
finally:
    print('finally here')
print('done')